### DISCUSSION TITLE
868Mhz LBT is the works?

### DISCUSSION THREAD

**Discussion by mha1** [ExpressLRS/ExpressLRS#2288] [General]
Hi,

Are there plans to make 868Mhz EU legal, i.e. LBT?

Thanks,
Michael

**Comment by mha1** [comment]
So I guess no

**Comment by FlyingBaguett3** [comment]
I'm having trouble finding a clear answer on regulations concerning the 868MHz band in the EU, but what I seem to find are restrictions based on power and duty cycle (I commonly find 1% duty cycle and 25mW), I don't see a mention of LBT.

**Comment by mha1** [comment]
maybe this helps to determine what's required: https://lora-developers.semtech.com/?ACT=72&fid=30&aid=48_0znCpZpvImL3agza59hG&board_id=1

**Comment by mha1** [comment]
I think LBT is hidden in the "polite spectrum access". Either LBT or the duty cycle requirement. I think LBT is the more performance oriented solution to EU compatibility.

**Comment by mha1** [comment]
[en_30022002v030201p-1.pdf](https://github.com/ExpressLRS/ExpressLRS/files/12129485/en_30022002v030201p-1.pdf)

We are limited to 25mW due to nandwidth requirements of our use case. If you want to go higher than the 1% duty cycle (865 MHz to 868 MHz) or  0,1% duty cycle (863 MHz to 865 MHz) polite spectrum access (= listen before talk) is required. Since we need to have more than 1% duty cycle for low latency (high update rates) LBT is a requirement. 

See para. 4.5.2.0 Applicability
Clear channel Assessment clause applies to EUT with polite spectrum access instead of duty cycle where permitted by
table B.1 in annex B, or table C.1 in annex C or any NRI.
 
Find table C1 on pages 22/23.

So are we going to implement LBT for 868Mhz?
